03/app.js

```diff
@@ -3,34 +3,66 @@ import ReactDOM from 'react-dom';
 
 class Article extends React.Component {
     state = {
+        comment: '',
         comments: [],
     }
     
     render() {
         const {title, body} = this.props;
+        const {comment} = this.state;
         return (
             <article>
                 <h1>{ title }</h1>
                 <p>{ body }</p>
                 <section>
-                    <form>
+                    <form onSubmit={ this.submitComment }>
                         <div>
                             <label>
                                 <textarea 
-                                    style={{ "min-width": "300px", "min-height": "120px" }} 
-                                    name="content" 
+                                    style={{ "minWidth": "300px", "minHeight": "120px" }} 
+                                    name="content"
+                                    value={ comment }
+                                    onChange = { this.commentChange }
                                 />
                             </label>
                         </div>
                         <div><input type="submit" value="dodaj komentarz" /></div>
                     </form>
                     <ul>
-                        {/* tutaj komentarze jako <li/>, ps. tak wygląda komentarz do kodu w JSX */}
+                        { this.updateCommentsList() }
                     </ul>
                 </section>
             </article>
         )
     }
+
+    commentChange = (e) => {     
+        this.setState({ comment : e.target.value })
+    }
+
+    submitComment = (e) => {
+        e.preventDefault();
+
+        const {comments, comment} = this.state;
+        
+        if (comment.trim()) {
+            const currComments = [...comments, comment];
```

Comment:
`commnet` to nie będzie "wyczyszczony" przez `trim()` bo ta funkcja zwraca nowy ciąg znaków - warto użyć dodatkowej zmiennej :)